packages/ui-components/src/components/UIInput/UITextInput.tsx

```diff
@@ -85,8 +85,7 @@ export class UITextInput extends React.Component<UITextInputProps> {
         return {
             ...{
                 root: {
-                    height: 'auto',
-                    fontFamily: 'var(--vscode-font-family)'
```

Comment:
as inputs don't inherit `font` from the parents, should we leave at least `font: inherit` here?

Reply:
As i checked, then we do not need it, fluent-ui still sets font-family to component itself, but instead of using default/fluent-ui font, then our font-family is used:
![image](https://github.com/user-attachments/assets/cd35f948-8688-425c-86eb-1a9c16082435)

Reply:
this is a div, not an input. 
But if the `input` has the same styling, than great :-) 
(just wanted to make sure)

Reply:
this is a div, not an input.
But if the input has the same styling, than great :-)
(just wanted to make sure)